exercises/concept/lasagna-master/.docs/instructions.md

```diff
@@ -70,8 +70,8 @@ The friend sent you the list of ingredients and told you the last item on the li
 Now you want to add that secret ingredient to your recipe as well.
 
 Write a function `addSecretIngredient` that accepts two arrays of ingredients as parameters.
-The first parameter is the list your friend sent you, the second is the ingredient list for your own recipe.
-The function should add the last item from your friends list to the end of your list.
+The first parameter is the list your friend sent you, the second is the ingredient list for your recipe.
```

Comment:
"own" seems to add value here, in the new version "your recipe" could also mean "your copy of the friend's recipe.